static/app/views/dashboards/widgetBuilder/components/datasetSelector.tsx

```diff
@@ -8,7 +8,7 @@ import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import useOrganization from 'sentry/utils/useOrganization';
 import {WidgetType} from 'sentry/views/dashboards/types';
-import {SectionHeader} from 'sentry/views/dashboards/widgetBuilder/components/sectionHeader';
+import {SectionHeader} from 'sentry/views/dashboards/widgetBuilder/components/utils/sectionHeader';
```

Comment:
imo `utils` are usually just helper functions that get pulled in, not really UI stuff. How do you feel about putting calling the `utils` folder `common`? so `sentry/views/dashboards/widgetBuilder/components/common/sectionHeader`

Reply:
ah yes! I like that better 😌